spec/features/customers/show_spec.rb

```diff
@@ -0,0 +1,103 @@
+
+    CustomerItem.create!(customer: @customer_1, item: @item_1)
+    CustomerItem.create!(customer: @customer_1, item: @item_2)
+    CustomerItem.create!(customer: @customer_1, item: @item_3)
+    CustomerItem.create!(customer: @customer_1, item: @item_4)
+
+    CustomerItem.create!(customer: @customer_2, item: @item_5)
+    CustomerItem.create!(customer: @customer_2, item: @item_6)
+    CustomerItem.create!(customer: @customer_2, item: @item_1)
+    CustomerItem.create!(customer: @customer_2, item: @item_2)
+
+    CustomerItem.create!(customer: @customer_3, item: @item_3)
+    CustomerItem.create!(customer: @customer_3, item: @item_4)
+    CustomerItem.create!(customer: @customer_3, item: @item_5)
+    CustomerItem.create!(customer: @customer_3, item: @item_6)
+
+  end
+
+  describe "US1 Show page" do 
+    it "displays a customer's name and the list of their items" do 
+      visit "/customers/#{@customer_1.id}"
+
+      within '#customer_name' do 
+        expect(page).to have_content("Customer Name: Jane K")
+      end
+
+      within '#item_list' do 
+        expect(page).to have_content("Banana, $1, Supermarket: Kroger")
+        expect(page).to have_content("Apple, $2, Supermarket: Kroger")
+        expect(page).to have_content("Pear, $3, Supermarket: Safeway")
+        expect(page).to have_content("Orange, $4, Supermarket: Sprouts")
+        expect(page).to_not have_content("Grapefruit, $5, Supermarket: Sprouts")
+      end
+      
+
+      visit "/customers/#{@customer_2.id}"
+      
+      within '#customer_name' do 
+        expect(page).to have_content("Customer Name: Rachel W")
+      end
+
+      within '#item_list' do 
+        expect(page).to have_content("Grapefruit, $5, Supermarket: Sprouts")
+        expect(page).to have_content("Avocado, $6, Supermarket: Safeway")
+        expect(page).to have_content("Banana, $1, Supermarket: Kroger")
+        expect(page).to have_content("Apple, $2, Supermarket: Kroger")
+        expect(page).to_not have_content("Orange, $4, Supermarket: Sprouts")
+      end
+
+      visit "/customers/#{@customer_3.id}"
+      
+      within '#customer_name' do 
+        expect(page).to have_content("Customer Name: Camila M")
+      end
+  
+      within '#item_list' do 
+        expect(page).to have_content("Pear, $3, Supermarket: Safeway")
+        expect(page).to have_content("Orange, $4, Supermarket: Sprouts")
+        expect(page).to have_content("Grapefruit, $5, Supermarket: Sprouts")
+        expect(page).to have_content("Avocado, $6, Supermarket: Safeway")
+        expect(page).to_not have_content("Apple, $2, Supermarket: Kroger")
+      end
+    end
+  end
+
+  describe "us2 - Form to add an item" do 
+    xit "displays a form for the customer to add a new item to their list" do 
+      visit "/customers/#{@customer_1.id}"
+
+      within '#add_item' do
+        expect(page).to have_field('search')
+
+        fill_in 'search', with: "Avocado"
```

Comment:
I think our form will want to be able to pass in an id of an existing item vs just the name